src/main/kotlin/no/nb/mlt/wls/application/synqapi/synq/SynqController.kt

```diff
@@ -52,13 +58,43 @@ class SynqController(
         )
     )
     @PutMapping("/item-update")
-    suspend fun updateItem(
+    suspend fun moveItem(
         @RequestBody synqBatchMoveItemPayload: SynqBatchMoveItemPayload
     ): ResponseEntity<Void> {
         synqBatchMoveItemPayload.mapToItemPayloads().map { moveItem.moveItem(it) }
         return ResponseEntity.ok().build()
     }
 
+    // TODO - more swagger docs
+    @Operation(
+        summary = "Confirms the picking of items from a specific SynQ order",
+        description = """Updates the items from a specific order when they are picked from a SynQ warehouse.
+            This does not update the order status, as SynQ sends an update to the order-update endpoint later.
+        """
+    )
+    @PutMapping("/pick-update/{owner}/{orderId}")
+    suspend fun pickOrder(
+        @Parameter(description = "Owner of the order items")
+        @PathVariable owner: Owner,
+        @Parameter(description = "Order ID in the storage system")
+        @PathVariable orderId: String,
+        @RequestBody payload: SynqOrderPickingConfirmationPayload
+    ) {
+        if (payload.orderLine.isEmpty()) throw ValidationException("Picking update does not contain any elements in the order line")
```

Comment:
I started moving this kind of validation to the payloads themselves. I think it's a bit more logical as it's payloads job to know if it's valid, not controller or other. If you look at what I have done in the recent commit to my branch you can see it for ApiOrderPayload and ApiItemPayload.

Reply:
I will take a look at that and see if i can't move this validation then